napari/settings/_base.py

```diff
@@ -364,8 +367,32 @@ def config_file_settings_source(
 
     # if the config has a `sources` list, read those too and merge.
     sources = list(getattr(settings.__config__, 'sources', []))
-    if config_path:
+    if config_path and Path(config_path).exists():
```

Comment:
You could make the `Path` object just once at the beginning.

Reply:
Hmm, i can't make this work because `config_path` may not exist, so the Path() is unbound.

Reply:
Got it. Use an `if` derp.